ios/Lift/LiftServer/SensorDataSourceLocation.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+/**
+ * Sensor data location matching the Scala codebase
+ */
+enum SensorDataSourceLocation : UInt8 {
+    case Wrist = 0x01
```

Comment:
This is why I was in favour of shared schema. Now these values are by convention so everyone implementing the encoder/decoder must know what 0x01 means and this code is duplicated in each device's code